app/hotels/src/singleHotel/summary/__tests__/RoomSummary.test.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import { RoomSummary } from '../RoomSummary';
+
+const room = {
+  availableRooms: [
+    {
+      id: '1',
+      selectedCount: 2,
+      incrementalPriceWithExtraCharges: [
+        {
+          price: {
+            currency: 'EUR',
+            amount: 5,
+          },
+        },
+        {
+          price: {
+            currency: 'EUR',
+            amount: 20,
+          },
+        },
+      ],
+      room: {
+        maxPersons: 2,
+      },
+    },
+    {
+      id: '2',
+      selectedCount: 1,
+      incrementalPriceWithExtraCharges: [
+        {
+          price: {
+            currency: 'EUR',
+            amount: 5,
+          },
+        },
+      ],
+      room: {
+        maxPersons: 2,
+      },
+    },
+    {
+      id: '3',
+      incrementalPriceWithExtraCharges: [
+        {
+          price: {
+            currency: 'EUR',
+            amount: 5,
+          },
+        },
+      ],
+      room: {
+        maxPersons: 2,
+      },
+    },
+  ],
+};
+
+describe('calculateSelectedData', () => {
+  it('calculates data correctly', () => {
+    // $FlowExpectedError: passing just props needed to run test
+    const Component = new RoomSummary({ room });
+
+    const test = Component.calculateSelectedData();
+    expect(test).toEqual({
+      amount: 25,
```

Comment:
Is this amount correct?
I would have thought that two rooms with (5+20) and another with (5), should be
2 * 25 + 1*5 = 55.

Reply:
No need to multiplicate, the prices under incrementalPrice[selectedCount - 1] is already calculated.